packages/shared/routes/onboarding/views/shimmer-claiming/views/ClaimRewardsView.svelte

```diff
@@ -31,7 +36,11 @@
         $shimmerClaimingRouter.previous()
     }
 
-    async function onUseBalanceFinderClick(): Promise<void> {
+    function _onLedgerNotConnected(): void {
+        displayNotificationForLedgerProfile('error', true)
+    }
+
+    async function onUseBalanceFinderClickInternal(): Promise<void> {
```

Comment:
I don't like appending `Internal` to a function name. Looking at it gives 0 clue what it does. I would name the function `searchShimmerRewards`